pkg/cmd/chat/user/user.go

```diff
@@ -296,3 +305,310 @@ func revokeCmd() *cobra.Command {
 
 	return cmd
 }
+
+func banCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ban-user --target-user-id [user-id] --banned-by-id [user-id] --reason [reason] --expiration [expiration-in-minutes]",
+		Short: "Ban a user",
+		Long: heredoc.Doc(`
+			Users can be banned from an app entirely.
+			When a user is banned, they will not be allowed to post messages until
+			the ban is removed or expired but will be able to connect to Chat
+			and to channels as before.
+
+			Channel watchers cannot be banned.
+		`),
+		Example: heredoc.Doc(`
+			# 'admin-user-1' bans user 'joe'
+			$ stream-cli chat ban-user --target-user-id joe --banned-by admin-user-1
```

Comment:
I would prefer to have commands sorted by category. Like `user-ban`, 'user-delete`. It would make them easy discoverable.

Reply:
Helps discoverability but the sentence "user-ban" makes less sense than "ban-user". Btw, we wanted to follow AWS CLI's pattern: https://docs.aws.amazon.com/cli/latest/reference/ecs/index.html#cli-aws-ecs

Reply:
😄 true